entities/src/models.rs

```diff
@@ -610,6 +619,26 @@ pub struct UnprocessedAccountMessage {
     pub id: String,
 }
 
+impl UnprocessedAccountMessage {
+    pub fn solana_change_info(&self) -> (Pubkey, u64, u64, u64) {
+        let (slot, write_version, data_hash) = match &self.account {
+            UnprocessedAccount::MetadataInfo(v) => (v.slot_updated, v.write_version, v.data_hash),
+            UnprocessedAccount::Token(v) => (v.slot_updated as u64, v.write_version, v.data_hash),
```

Comment:
Are we ok with converting `i64` as `u64`? In case it cannot be negative, why then `TokenAccount` stores it as i64? Some kind of restrictions from the DB?